amgut/db/patches/0038.sql

```diff
@@ -0,0 +1,362 @@
+(190, 'PM_HEALTH', 'How good is your health?', 'How good is your health?'),
+(191, 'PM_PARTICIPATION_REASON', 'What prompted you to participate in the American Gut Project? Please select all that apply.', 'What prompted you to participate in the American Gut Project? Please select all that apply.'),
+(192, 'PM_GASTRO_PROBLEMS', 'What type of problems do you have? Please select all that apply.', 'What type of problems do you have? Please select all that apply.'),
+(193, 'PM_UNDERSTAND_RESULTS', 'Do you feel you understand your personal microbiome results?', 'Do you feel you understand your personal microbiome results?'),
+(194, 'PM_USEFUL', 'In general, do you feel that your personal microbiome results are useful to you?', 'In general, do you feel that your personal microbiome results are useful to you?'),
+(195, 'PM_LIFESTYLE_CHANGE', 'Do you think receiving your personal microbiome results has changed your behavior or lifestyle in any way?', 'Do you think receiving your personal microbiome results has changed your behavior or lifestyle in any way?'),
+(196, 'PM_LIFESTYLE_CHANGE_HOW', 'In what way has receiving your personal microbiome results changed your behavior or lifestyle? Please select all that apply.', 'In what way has receiving your personal microbiome results changed your behavior or lifestyle? Please select all that apply.'),
+(197, 'PM_CONCERN', 'Now that you have your personal microbiome results, OVERALL are you', 'Now that you have your personal microbiome results, OVERALL are you'),
+(198, 'PM_SHARED', 'Have you shared or discussed your personal microbiome results with anyone else?', 'Have you shared or discussed your personal microbiome results with anyone else?'),
+(199, 'PM_SHARED_WHO', 'With whom did you share or discuss your personal microbiome results? Please select all that apply.', 'With whom did you share or discuss your personal microbiome results? Please select all that apply.'),
+(200, 'PM_SHARED_PCP', 'If you shared or discussed your personal microbiome results with your primary care physician or another health professional, how willing was s/he to discuss your results with you?', 'If you shared or discussed your personal microbiome results with your primary care physician or another health professional, how willing was s/he to discuss your results with you?'),
+(201, 'PM_SHARED_PCP_CHANGES', 'If you shared your personal microbiome results with your primary care physician or another health professional, did s/he make any changes to your healthcare?', 'If you shared your personal microbiome results with your primary care physician or another health professional, did s/he make any changes to your healthcare?'),
+(202, 'PM_SHARED_PCP_CHANGES_WHAT', 'What changes did your healthcare provider make to your healthcare? Please select all that apply.', 'What changes did your healthcare provider make to your healthcare? Please select all that apply.'),
```

Comment:
It's a triggered question, so the only way they see it is if the user responded yes to the previous question. Seemed redundant because of that, but I can put the wording back in if needed.